mozilla_schema_generator/__main__.py

```diff
@@ -214,16 +214,21 @@ def generate_glean_pings(config, out_dir, split, pretty, repo, generic_schema):
 
 
 def write_schema(repo, repo_id, config, out_dir, split, pretty, generic_schema):
-    schema_generator = GleanPing(repo)
+    schema_generator = GleanPing(repo, repo_id)
```

Comment:
We need GleanPing to keep track of the repo_id (app_id) now since the generated metadata depends on it for the bq_dataset_family param.